distributionviewer/core/templates/distributionviewer/login.html

```diff
@@ -7,6 +7,14 @@
     <meta name="google-signin-client_id" content="{{ google_clientid }}">
     <title>Firefox Distribution Viewer</title>
     <link rel="stylesheet" type="text/css" href="{% static 'css/bundle.css' %}">
+    {% if user.is_authenticated %}
```

Comment:
I was wondering about that. We could definitely special-case this for users who aren't able to access the */admin/* page. On the other hand, by checking only whether the user is authenticated, we can infer that they lacked permission to the intended page for any reason.

I used [this permission error message](https://github.com/django/django/blob/master/django/contrib/admin/templates/admin/login.html#L35-L42) as inspiration.